### DISCUSSION TITLE
compile error when I migrate to 2.0.0

### DISCUSSION THREAD

**Discussion by jxnu-liguobin** [zio/zio#7029] [Q&A]
I migrated from 2.0.0RC6 to 2.0.0

many errors：
could not find implicit value for parameter trace: zio.Trace

**Comment by adamgfraser** [comment]
Can you post a small reproducer? Traces should be automatically generated.

**Reply by jxnu-liguobin** [reply]
Thank you for your reply, But I reinstalled the OS today and now I don't see the error. 😅
